docs/config.json

```diff
@@ -494,6 +494,14 @@
             }
           ]
         },
+        {
+          "title": "Access Monitoring",
+          "slug": "/access-controls/access-monitoring/",
+          "forScopes": [
+            "team",
+            "cloud"
+          ]
```

Comment:
We should also add a guide explaining how to set it up for self-hosted clusters. We can do it as a follow-up to this PR though so we can deploy these docs quicker.